github/ci/prow/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-postsubmits.yaml

```diff
@@ -25,6 +25,8 @@ postsubmits:
               value: "../project-infra/hack/git-askpass.sh"
             - name: GITHUB_USER
               value: kubevirt-bot
+            - name: GITHUB_EMAIL
```

Comment:
isn't it GIT_AUTHOR_EMAIL?
at least here
https://github.com/kubevirt/project-infra/blob/324bf299b0ce20a1af359e6317430d0d96d3d8f5/github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml#L170

unless there are couple tuples that would work